pysisyphus/helpers.py

```diff
@@ -51,7 +51,8 @@ def geom_from_xyz_file(xyz_fn, coord_type="cart", **coord_kwargs):
     kwargs.update(coord_kwargs)
     xyz_fn = str(xyz_fn)
     atoms, coords, comment = parse_xyz_file(xyz_fn, with_comment=True)
-    coords *= ANG2BOHR
+    if "IN_BOHR" not in comment.split():
```

Comment:
Please also add a comment that highlights again, that the produced `xyz`-string will usually be in Angstrom and only when `IN_BOHR` is present, it will be in Bohr.